client/packages/invoices/src/Returns/modals/SupplierReturn/SupplierReturn.tsx

```diff
@@ -76,8 +78,9 @@ export const SupplierReturnEditModal = ({
 
   const onOk = async () => {
     try {
-      !isDisabled && (await save());
+      let supplierReturn = !isDisabled && (await save());
       onCreate?.();
+      !!supplierReturn && supplierReturn?.originalShipment?.id && isNewReturn && success(t('messages.supplier-return-created-shipped'))();
```

Comment:
See above comment

Reply:
This one is needed because the same insert modal is used both from appbarbuttons in the return list views, and in shipment list views. And in both cases a return node is returned - so we need the additional check that it has an original shipment id.